ror-tools/src/test/scala/tech/beshu/ror/tools/RorToolsAppSuite.scala

```diff
@@ -496,12 +440,14 @@ class RorToolsAppSuite
            | - file x-pack-core-$esVersionUsed.jar was patched by ROR ${metadata.rorVersion}
            | - file x-pack-ilm-$esVersionUsed.jar was patched by ROR ${metadata.rorVersion}
            | - file x-pack-security-$esVersionUsed.jar was patched by ROR ${metadata.rorVersion}
-           |""".stripMargin
+           |""".stripMargin.replace("\r\n", "\n")
       )
     }
     "Successfully patch, verify and unpatch" in {
+      // This file is created after patching on Windows for ES 8.x
+      val filesExcludedFromHashCalculation = List("plugin-security.policy.tmp")
```

Comment:
Seems to be some ES 8.x issue specific to Windows. It does not happen on other ES versions and on Linux.